### ISSUE TITLE
about results fo regDB

### ISSUE THREAD

**Issue by FranklinLF** [yangbincv/ADCA#2]
Do you need to modify the clustering parameters when using the regDB dataset？

**Comment by yangbincv** [comment]
DBSCAN(eps=0.3, min_samples=4, metric='precomputed', n_jobs=-1) for ir and rgb data in regDB dataset.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "yangbincv/ADCA"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年2月23日(星期四) 下午5:16
***@***.***&gt;;
***@***.***&gt;;
主题:&nbsp;[yangbincv/ADCA] about results fo regDB (Issue #2)





 
Do you need to modify the clustering parameters when using the regDB dataset？

**Comment by FranklinLF** [comment]
Is channeladapgray for infrared images used in the implementation of the algorithm?

**Comment by yangbincv** [comment]
Yes,ChannelAdapGray(probability =0.5) is used in&nbsp;train_loader_ir



------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年2月23日(星期四) 晚上6:56
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [yangbincv/ADCA] about results fo regDB (Issue #2)





 
Is channeladapgray for infrared images used in the implementation of the algorithm?
